Source/relay/TourGuide/Pulls.ash

```diff
@@ -489,13 +489,14 @@ void generatePullList(Checklist [int] checklists)
             pullable_item_list.listAppend(GPItemMake($item[xiblaxian stealth vest], "-combat shirt"));
         if ($item[duonoculars].item_is_usable())
 	        pullable_item_list.listAppend(GPItemMake($item[duonoculars], "-combat, +5 ML"));
-        pullable_item_list.listAppend(GPItemMake($item[ring of conflict], "-combat"));
+        if ($item[ring of conflict].item_is_usable())
+            pullable_item_list.listAppend(GPItemMake($item[ring of conflict], "-combat"));
```

Comment:
actually, speaking of which, it looks like this code doesn't actually handle TCRS at all; any equipment pull recs for 2CRS that are pull ideas dependent on enchantments are just straight broken. not relevant to this PR but adding this note to issue #40